tests/Tokens/setComptrollerTest.js

```diff
@@ -16,21 +16,22 @@ describe('CToken', function () {
 
   describe('_setComptroller', () => {
     it("should fail if called by non-admin", async () => {
-      expect(
-        await send(cToken, '_setComptroller', [newComptroller._address], { from: accounts[0] })
-      ).toHaveTokenFailure('UNAUTHORIZED', 'SET_COMPTROLLER_OWNER_CHECK');
+      await expect(send(cToken, '_setComptroller', [newComptroller._address], { from: accounts[0] }))
```

Comment:
Codacy has a fix for the issue: [Replace `send(cToken,·'_setComptroller',·[newComptroller._address],·{·from:·accounts[0]·})` with `⏎········send(cToken,·"_setComptroller",·[newComptroller._address],·{⏎··········from:·accounts[0],⏎········}`](https://app.codacy.com/gh/cloudwalk/compound-protocol/pullRequest?prid=9803552)
```suggestion
      await expect(
```